src/components/Events/Events.tsx

```diff
@@ -20,25 +20,48 @@ type EventsPropTypes = any;
 @translate(['events'], { wait: true })
 export default class Events extends Component<EventsProps, EventsPropTypes> {
 	public static propTypes = {
-		nextEvent: PropTypes.object.isRequired,
-		pastEvents: PropTypes.array.isRequired,
+		nextMeetupEvents: PropTypes.array.isRequired,
+		noNextMeetupEvent: PropTypes.bool.isRequired,
+		pastMeetupEvents: PropTypes.array.isRequired,
 	};
 
 	public render() {
-		const { t, nextEvent, pastEvents } = this.props;
-		const pastEventsList = pastEvents.map((mEvent: any) => (
+		const {
+			t,
+			nextMeetupEvents,
+			noNextMeetupEvent,
+			pastMeetupEvents,
+		} = this.props;
+		const nextEventsList = nextMeetupEvents.map(
+			({ description, withDescription, ...restOfEvent }: any) => (
+				// Strip the description and withDescription properties
+				<EventCard key={restOfEvent.time} MeetupEvent={...restOfEvent} />
+			)
+		);
+		const nextEventsBlock =
+			nextMeetupEvents && nextMeetupEvents.length > 0 ? (
+				<Fragment>
```

Comment:
Instead of using `<Fragment> ... </Fragment>` you could also use `<> ... </>`. This might be a benefit of Typescript but I'm not sure.